lib/ansible/modules/expect.py

```diff
@@ -159,17 +206,21 @@ def main():
     creates = module.params['creates']
     removes = module.params['removes']
     responses = module.params['responses']
+    raw_responses = module.params['raw_responses']
     timeout = module.params['timeout']
     echo = module.params['echo']
 
+    # one between responses and raw_responses needs to be specified
+    if responses is None and raw_responses is None:
+        module.fail_json(msg="responses (or raw_responses) is required")
```

Comment:
Instead of this block, use `required_one_of` for `AnsibleModule`:

```python
module = AnsibleModule(
    argument_spec=...,
    required_one_of=[['responses', 'raw_responses']],
)
```

Reply:
Thank you @sivel for the feedback, I have updated the PR to implement your suggestion!